make/validate.mk

```
@@ -8,4 +8,4 @@ $(DKP_BLOODHOUND_BIN):
 
 .PHONY: validate-manifests
 validate-manifests: $(DKP_BLOODHOUND_BIN)
-	$(DKP_BLOODHOUND_BIN) . --helm-repo-path ./helm-repositories
+	$(DKP_BLOODHOUND_BIN)
```

```suggestion
	$(DKP_BLOODHOUND_BIN) -v 4
```

or some appropriate number - right now, CI does not print anything informative.